src/contribute-to-tidb/miscellaneous-topics.md

```diff
@@ -0,0 +1,12 @@
+# Miscellaneous Topics
+
+## Communication channels
+
+- [TiDB Internals Forum](https://internals.tidb.io): TiDB hosts a Discourse instance for TiDB development discussions. It has many separate categories for different topics. You can discuss anything about TiDB development and community in the forum.
+
+## Related projects
+
+- [TiKV](https://github.com/tikv/tikv): TiKV is an open-source, distributed, and transactional key-value database. It is used by TiDB as the storage layer.
+- [Talent Plan](https://university.pingcap.com/talent-plan/): Talent Plan is an open source training program initiated by PingCAP. It aims to create or combine some open source learning materials for people interested in open source, distributed systems, Rust, Golang, and other infrastructure knowledges.
```

Comment:
```suggestion
- [Talent Plan](https://university.pingcap.com/talent-plan/): Talent Plan is an open source training program initiated by PingCAP. It aims to create and combine some open source learning materials for people interested in open source, distributed systems, Rust, Golang, and other infrastructure knowledges.
```

Reply:
I +1 for `and` over `or`. I copied this introduction from https://github.com/pingcap/talent-plan .